api/app/Console/Commands/Search/SetUpSearch.php

```diff
@@ -5,28 +5,31 @@
 namespace App\Console\Commands\Search;
 
 use Illuminate\Console\Command;
-use MeiliSearch\Client as Meilisearch;
+use Meilisearch\Client as Meilisearch;
 
 class SetUpSearch extends Command
 {
     protected $signature = 'search:setup {--reset} {--import}';
 
     protected $description = 'Setup search indices, settings, etc.';
 
-    public function handle(Meilisearch $client): int
+    private Meilisearch $client;
+
+    public function handle(): int
     {
+        $this->client = new Meilisearch('http://search:7700', 'secret');
```

Comment:
This isn't the right approach. I see a couple options.

1) Create a binding in `AppServiceProvider` that resolves `Meilisearch\Client::class` and constructs it properly. Something like:
```php
$this->app->singleton(\Meilisearch\Client::class, function () {
    return new \Meilisearch\Client(config('scout.meilisearch.host'), config('scout.meilisearch.key'));
});
```

Feel free to clean this up as you wish.

2) Upgrade to a newer Laravel Scout that supports this version of meilisearch natively.